[8주차]Cow_mvc_practice(win9-tech)

win9-tech opened the pull request:
https://documenter.getpostman.com/view/32531840/2sA3Qngt8f

win9-tech committed:
[refactor] 패지키 구조 리팩토링

Hoya324 commented on src/main/java/com/cow/cow_mvc_practice/member/controller/MemberController.java:
```diff
@@ -23,26 +17,19 @@ public class MemberController {
 	/* MemberResponse dto 적용 */
 	@PostMapping("/new")
 	public MemberResponse create(@RequestBody final MemberRequest memberRequest) {
+		System.out.println("111");
```
test 한 코드는 지워주세요!

Hoya324 reviewed (commented):
과제가 1주 남긴했지만 꾸준히 올리면 더 좋은 코드를 작성할 수 있습니다..! 파이팅

win9-tech committed:
[Feat] 게시글 생성 구현

win9-tech committed:
[Feat] 전체 게시글조회 구현

win9-tech committed:
[Feat] 사용자 정보 수정 구현

win9-tech committed:
[Feat] 단일 게시글 조회 구현

win9-tech committed:
[Feat] 댓글 작성 구현

win9-tech committed:
[Feat] 게시글 삭제 구현

win9-tech committed:
[refactor] dto패키지 위치 변경

win9-tech committed:
[refactor] Member 생성 코드 개선

win9-tech committed:
[refactor] Member update 기능 개선

UpdateMemberRequest 적용
UpdateMemberResponse 적용
update 흐름 변경 service -> request -> entity

win9-tech committed:
[refactor] Member find 기능 개선

FindMemberResponse 적용

win9-tech committed:
[refactor] All Member find 기능 개선

FindMemberResponse 적용
@Transactional(readOnly = true) 적용

win9-tech committed:
[refactor] CreateMemberRequest toEntity 메서드 Member entity 정적 팩토리메서드 사용

win9-tech committed:
[refactor] MemberController, MemberService 메서드명 수정

win9-tech committed:
[refactor] PostController, PostService 메서드명 수정

win9-tech committed:
[refactor] Post delete 반환타입 변경

win9-tech committed:
[refactor] Comment 생성 코드 개선

win9-tech committed:
[refactor] 코드 컨벤션 수정

win9-tech committed:
[refactor] baseEntity, TimestampedEntity 적용

win9-tech committed:
[refactor] 연관관계 편의 메서드 추가, Dto 구조 변경

win9-tech committed:
[refactor] CreatedCommentResponse 컨벤션 수정

win9-tech committed:
database erd 추가

BaekJaehyuk commented on src/main/java/com/cow/cow_mvc_practice/comment/dto/response/CreatedCommentResponse.java:
```diff
@@ -0,0 +1,35 @@
+    private final String content;
+    private final LocalDateTime createTime;
+
+    @Builder
+    private CreatedCommentResponse(final Long id, final Long postId, final Long memberId, final String content, final LocalDateTime createTime) {
+        this.id = id;
+        this.postId = postId;
+        this.memberId = memberId;
+        this.content = content;
+        this.createTime = createTime;
+    }
```
이거는 어노테이션으로 처리해도 괜찮을 것 같습니다 : )

baekjaehyuk reviewed (commented):
LGTM ~ ★

KoSeonJe reviewed (commented):
LGTM ~ ★
